apps/posts/views.py

```diff
@@ -27,3 +27,8 @@ def get_permissions(self):
         elif self.action in ["destroy", "update"]:
             self.permission_classes = [IsAuthor]
         return super().get_permissions()
+    
+    def get_queryset(self):
+        queryset = self.queryset
```

Comment:
상속 받아서 쓰는게 좋을 것 같습니다.
```suggestion
        queryset = super().get_queryset()
```